tests/testthat/_snaps/geodist.md

```diff
@@ -162,6 +162,41 @@
       Error in `step_geodist()`:
       ! `is_lat_lon` should be a single logical value.
 
+# empty printing
+
+    Code
+      rec
```

Comment:
What do you think about adding the definition of `rec` to the snapshots? Then we could see that it's testing the right thing without referring to the `test-*.R` files.

Reply:
it is not a bad idea. 

I think I like to stay with the current design, as all the steps follow this formula with a couple of small variations.

```r
test_that("empty printing", {
  rec <- recipe(mpg ~ ., mtcars)
  rec <- name_of_step_here(rec)

  expect_snapshot(rec)

  rec <- prep(rec, mtcars)

  expect_snapshot(rec)
})

```